src/helpers.js

```diff
@@ -3,30 +3,50 @@
 import FS from 'sb-fs'
 import Path from 'path'
 import semver from 'semver'
-import { exec } from 'sb-exec'
+import { BufferedProcess } from 'atom'
 import type { Dependency } from './types'
 
 let shownStorageInfo = false
 const VALID_TICKS = new Set(['✓', 'done'])
 const VALIDATION_REGEXP = /(?:Installing|Moving) (.*?) to .* (.*)/
 
+export function exec(command: string, parameters: Array<string>): Promise<{ stdout: string, stderr: string }> {
+  return new Promise(function(resolve) {
+    const data = { stdout: [], stderr: [] }
+    const spawnedProcess = BufferedProcess({
+      command,
+      args: parameters,
+      stdout(chunk) {
+        data.stdout.push(chunk)
+      },
+      stderr(chunk) {
+        data.stderr.push(chunk)
+      },
+      exit() {
+        resolve({ stdout: data.stdout.join(''), stderr: data.stderr.join('') })
+      },
+      autoStart: false,
```

Comment:
This is for the ESLint rule no sideeffect constructor, because that way we would only be doing `new BufferedProcess()` and that is wrong, because we won't be using that instance anywhere